<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleDBWidget</class>
 <widget class="QWidget" name="ModuleDBWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
      <widget class="QTabWidget" name="tabWidget">
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Module Inventory</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Filters:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Layer type:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="layertypeCB">
               <item>
                <property name="text">
                 <string>any</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Speed:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="speedCB">
               <item>
                <property name="text">
                 <string>any</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5G</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10G</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Spacer:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="spacerCB">
               <item>
                <property name="text">
                 <string>any</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4.0mm</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2.6mm</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1.8mm</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Grade:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="spacerCB_2">
               <item>
                <property name="text">
                 <string>any</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Status:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="spacerCB_3">
               <item>
                <property name="text">
                 <string>Ready For Mounting</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mounted</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>To Be Tested</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Inventory Slot</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Speed</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Spacer</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Status</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Description</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Connections</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Mounted_on</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="selectModulePB">
              <property name="text">
               <string>&lt;-- Select Module</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="viewDetailsPB">
              <property name="text">
               <string>View Details --&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="moduleDetailsTab">
         <attribute name="title">
          <string>Module Details</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="moduleNameLabel">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="detailsTree">
            <property name="columnCount">
             <number>2</number>
            </property>
            <column>
             <property name="text">
              <string>Field</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="editDetailsButton">
              <property name="text">
               <string>Edit Selected</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveDetailsButton">
              <property name="text">
               <string>Save Changes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="disconnectButton">
              <property name="text">
               <string>Disconnect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
      </widget>
    </item>
  </layout>
 </widget>
</ui> 